examples/custom_design/designs/core_site/jobs.py

```diff
@@ -28,3 +28,6 @@ class Meta:
         commit_default = False
         design_file = "designs/0001_design.yaml.j2"
         context_class = CoreSiteContext
+        version = "0.2.5"
```

Comment:
This is new... are we leveraging the design version yet?

Reply:
only informational for now, but not used to update the design yet.